### PR TITLE
Take distro (to test build on) from user's config

### PR DESCRIPTION
**Pull request by jpopelka**
This is the first part of #1230
The 1-to-n mapping will be in a separate commit since it will be much more work (and nobody needs it at the moment).

Related to: #1230 

Merge after: packit/packit#1407

---

It is possible to specify in the packit config which distro to test a build on in Testing Farm.

### CODE REVIEW

**Comment by jpopelka** [comment]
Tests are expected to fail now until packit/packit#1407 is merged.

File: packit_service/worker/testing_farm.py [hunk 1]
```diff
@@ -207,15 +207,19 @@ def chroot2distro_arch(self, chroot: str) -> Tuple[str, str]:
...
         distro, arch = chroot.rsplit("-", 1)
 
-        epel_mapping = {
-            "epel-6": "centos-6",
-            "epel-7": "centos-7",
-        }
+        distros = self.job_config.metadata.targets_dict.get(chroot, {}).get("distros")
+        if distros:
+            # For now take just the first one
+            return distros[0], arch
```
Comment (lachmanfrantisek) [inline packit_service/worker/testing_farm.py:213]: What about [assignment expression](https://www.python.org/dev/peps/pep-0572/)?

```suggestion
        if distros := self.job_config.metadata.targets_dict.get(chroot, {}).get("distros")
            # For now take just the first one
            return distros[0], arch
```

**Review by lachmanfrantisek — APPROVED** [review]
Looks good!

File: packit_service/worker/testing_farm.py (hunk 1 above)
Comment (jpopelka) [reply packit_service/worker/testing_farm.py:213]: Ah, actually black reformatted it into
```python
        if distros := self.job_config.metadata.targets_dict.get(chroot, {}).get(
            "distros"
        ):
```
in which case I think the old
```python
        distros = self.job_config.metadata.targets_dict.get(chroot, {}).get("distros")
        if distros:
```
is a bit nicer and easier to "parse". So I'd probably leave it as it is.

**Comment by jpopelka** [comment]
recheck

**Pull request merged**